### PR TITLE
Add Content-Security-Policy in PHP so we can have unsafe-eval when th…

### PR DESCRIPTION
**Pull request by nickygerritsen**
…e profiler is enabled

### CODE REVIEW

**Review by thijskh — APPROVED** [review]

**Comment by mvr320** [comment]
Why can you not do this in the configure step?

You only need this for maintainer-install right?

**Comment by nickygerritsen** [comment]
> Why can you not do this in the configure step?
> 
> You only need this for maintainer-install right?

You can switch to dev/prod mode every easily in any install by changing one line in `.env.local` (and i sometimes do this to debug stuff)

**Comment by thijskh** [comment]
I'm not sure maintainer-install and running in dev mode are necessarily the same thing?

**Comment by mvr320** [comment]
> 
> 
> > Why can you not do this in the configure step?
> > You only need this for maintainer-install right?
> 
> You can switch to dev/prod mode every easily in any install by changing one line in `.env.local` (and i sometimes do this to debug stuff)

Everyday I learn something new. But than I do indeed agree your solution is better/actually good

File: webapp/src/EventListener/AddContentSecurityPolicyListener.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        return [ResponseEvent::class => 'onKernelResponse'];
+    }
+
+    public function onKernelResponse(ResponseEvent $event)
+    {
+        // Set the correct CSP based on whether the profiler is enabled, since
+        // the profiler requires 'unsafe-eval' for script-src 'self'
+        $response = $event->getResponse();
+        $cspExtra = $this->profiler ? " 'unsafe-eval'" : "";
+        $csp = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline'$cspExtra; img-src 'self' data:";
```
Comment (meisterT) [inline webapp/src/EventListener/AddContentSecurityPolicyListener.php:35]: Can we add a space between `'` and `$` for readability?

Comment (nickygerritsen) [reply webapp/src/EventListener/AddContentSecurityPolicyListener.php:35]: Done

**Pull request merged**